oaknational/Oak-Web-Application
# style(reset-button): default 0 margin on button element

## Description

- "Manage cookie settings" lines up withother links in footer in safari

## Issue(s)

Fixes #709

## How to test

1. Go to {cloud link} in macOs safari
3. You should see in footer that the links all line up


## Checklist
